### ISSUE TITLE
One GenericRA deployment should work with more than one JMS provider

### ISSUE THREAD

**Issue by arinban** [arindam-bandyopadhyay/issue-test-new#4]
<a href="https://github.com/arinban"><img src="https://avatars3.githubusercontent.com/u/36043680?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [arinban](https://github.com/arinban)**
_Thursday Feb 01, 2018 at 20:21 GMT_
_Originally opened as https://github.com/arindam-bandyopadhyay/issue-test/issues/4_

----

One major usability requirement for generic ra is the ability to work with
multiple JMS providers with just only one RA deployment.

In short a J2EE App should be able to lookup connection factories send and
receive messages to more than one provider from one deployment of the genericra.
#### Environment
Operating System: All
Platform: All
#### Affected Versions
[0.9]

**Comment by arinban** [comment]
<a href="https://github.com/arinban"><img src="https://avatars3.githubusercontent.com/u/36043680?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [arinban](https://github.com/arinban)**
_Thursday Feb 01, 2018 at 20:21 GMT_

----

*Original Issue:https://github.com/javaee/glassfish-genericjmsra/issues/4
*Raised By:@glassfishrobot
*Created at:Tue Jun 07 08:05:42 IST 2005
*Assigned To:@glassfishrobot

**Comment by arinban** [comment]
<a href="https://github.com/arinban"><img src="https://avatars3.githubusercontent.com/u/36043680?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [arinban](https://github.com/arinban)**
_Thursday Feb 01, 2018 at 20:21 GMT_

----

@glassfishrobot Commented on Tue Jun 07 08:05:42 IST 2005
Reported by binod

**Comment by arinban** [comment]
<a href="https://github.com/arinban"><img src="https://avatars3.githubusercontent.com/u/36043680?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [arinban](https://github.com/arinban)**
_Thursday Feb 01, 2018 at 20:21 GMT_

----

@glassfishrobot Commented on Tue Jun 07 08:05:42 IST 2005
Was assigned to v_narayanan

**Comment by arinban** [comment]
<a href="https://github.com/arinban"><img src="https://avatars3.githubusercontent.com/u/36043680?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [arinban](https://github.com/arinban)**
_Thursday Feb 01, 2018 at 20:22 GMT_

----

@glassfishrobot Commented on Mon Apr 24 13:01:08 IST 2017
This issue was imported from java.net JIRA GENERICJMSRA-4
